editor/src/main/java/net/rpgtoolkit/editor/editors/TileSetCanvas.java

```diff
@@ -52,7 +52,7 @@ public TileSetCanvas(TileSet tileset) {
 
     this.tileset = tileset;
     int width = 320;

```

Comment:
This can probably be tileset.getTileWidth() * TILES_PER_ROW. I'm not sure how the width and height defined here interact with other things like getPreferredScrollableViewportSize() though. Nor do I know what would happen if you opened a tileset that had a high enough tile width that you couldn't fit 10 of them on the screen.